src/features/user/ManageProjects/components/DetailedAnalysis.tsx

```diff
@@ -285,7 +274,7 @@ export default function DetailedAnalysis({
       })
       .catch((err) => {
         setIsUploadingData(false);
-        setErrorMessage(err);
+        setErrorMessage('Internal server error');
```

Comment:
@sunilsabatp Not all errors will be of type 500. Doing this will not give us enough information about the error